emrun.py

```diff
@@ -1708,6 +1708,14 @@ def run(cmd):
     # if browser[0] == 'cmd':
     #   Workaround an issue where passing 'cmd /C start' is not able to detect when the user closes the page.
     #   serve_forever = True
+
+    try:
+      # Check if psutil can be imported, as waiting for browser pid silently fails otherwise.
```

Comment:
Would we instead modify the waiting code to not silently fail?  i.e. move this warning there?

Reply:
I decided against this, because the waiting code is being called multiple times. This would either mean we get the same warning multiple times or we had to remember that this warning was emitted earlier.

Reply:
Its just that this try/catch block is now duplicated many times within this code.

How about this: at the top level of the module we do this try catch exactly once and set `psutil = None` in the catch block.

Then anywhere that wants to check for it can just do `if psutil:`?

My understands is that function-level imports are bad form anyway so it would be a win-win.